shell/aoe_translate.php

```diff
@@ -50,6 +56,57 @@ public function generateStoreTranslationsActionHelp()
     }
 
     /**
+     * @return void
+     */
+    public function exportTranslationTableAction()
+    {
+        $resourceCollection = Mage::getResourceModel('aoe_translate/log_collection');
+        foreach ($resourceCollection as $item) {
```

Comment:
You could probably better do this via getData() and using the array since we don't need the object for each row.